source/source-example.yaml

```diff
@@ -6,6 +6,9 @@ metadata:
 spec:
   brokers: RABBITMQ_BROKERS
   topic: RABBITMQ_TOPIC
+  retry: 5
+  backoffPolicy: "linear"
+  backoffDelay: "PT30S"
```

Comment:
Does the period format "PT30S" work with time.Duration?

Reply:
You are right let me change it